config-model/src/main/java/com/yahoo/searchdefinition/processing/ImplicitSummaries.java

```diff
@@ -114,7 +113,7 @@ private void collectSummaries(SDField field , Schema schema, boolean validate) {
     }
 
     private DocumentSummary getOrCreateAttributePrefetchSummary(Schema schema) {
-        DocumentSummary summary = schema.getSummary("attributeprefetch");
+        DocumentSummary summary = schema.getSummariesInThis().get("attributeprefetch");
```

Comment:
attributeprefetch not dead yet.....

Reply:
Somebody needs to work on that and the "default" nonsense if we're to make it in time for Vespa 8 ...